### PR TITLE
Mount log dir for ncp/nsx-node-agent/nsx-ncp-bootstrap

### PR DESCRIPTION
**Pull request by heypnus**

### CODE REVIEW

File: manifest/openshift4/coreos/cluster-network-22-nsx-node-agent.yaml [hunk 1]
```diff
@@ -135,4 +138,6 @@ spec:
         name: dbus
       - hostPath: {path: /etc/sysconfig/network-scripts}
         name: nscripts
+      - hostPath: {path: /var/log/nsx-ujo}
```
Comment (dantingl) [inline manifest/openshift4/coreos/cluster-network-22-nsx-node-agent.yaml:141]: Looks like it is already in line 129

Comment (heypnus) [reply manifest/openshift4/coreos/cluster-network-22-nsx-node-agent.yaml:141]: the host-var-log-ujo in line 129 is mounted as /var/log/openvswitch, you can see line 86

File: manifest/kubernetes/rhel/cluster-network-32-nsx-node-agent.yaml [hunk 2]
```diff
@@ -132,4 +135,6 @@ spec:
         name: dbus
       - hostPath: {path: /etc/sysconfig/network-scripts}
         name: nscripts
+      - hostPath: {path: /var/log/nsx-ujo}
```
Comment (ggverma) [inline manifest/kubernetes/rhel/cluster-network-32-nsx-node-agent.yaml:138]: same as Danting's comment. volume already exists

File: manifest/kubernetes/ubuntu/cluster-network-32-nsx-node-agent.yaml [hunk 3]
```diff
@@ -122,4 +125,6 @@ spec:
         name: host-os-release
       - hostPath: {path: /var/log/nsx-ujo, type: DirectoryOrCreate}
         name: host-var-log-ujo
+      - hostPath: {path: /var/log/nsx-ujo}
```
Comment (ggverma) [inline manifest/kubernetes/ubuntu/cluster-network-32-nsx-node-agent.yaml:128]: here as well

File: manifest/openshift4/coreos/cluster-network-22-nsx-node-agent.yaml (hunk 1 above)
Comment (heypnus) [reply manifest/openshift4/coreos/cluster-network-22-nsx-node-agent.yaml]: removed the dup.

File: manifest/kubernetes/rhel/cluster-network-32-nsx-node-agent.yaml (hunk 2 above)
Comment (heypnus) [reply manifest/kubernetes/rhel/cluster-network-32-nsx-node-agent.yaml]: done, thanks

**Review by ggverma — APPROVED** [review]

**Review by dantingl — APPROVED** [review]

**Pull request merged**